### DISCUSSION TITLE
How to resovle the problem

### DISCUSSION THREAD

**Discussion by arde2020** [Chia-Network/chia-blockchain#5055] [Q&A]
when I install timerlord I have this problem，I use ubnutu 18.04 ,thanks

> Building wheels for collected packages: chiavdf
>   Building wheel for chiavdf (PEP 517) ... error
>   ERROR: Command errored out with exit status 1:
>    command: /service/sun/chia-blockchain/venv/bin/python /service/sun/chia-blockchain/venv/lib/python3.7/site-packages/pip/_vendor/pep517/in_process/_in_process.py build_wheel /ipfs/tmpdir/tmp17ya3y2u
>        cwd: /ipfs/tmpdir/pip-install-jgg5wz7d/chiavdf_ef617ebcf39f4fb4bd1906d66b679e00
>   Complete output (69 lines):
>   running bdist_wheel
>   running build
>   running build_ext
>   -- The C compiler identification is GNU 7.5.0
>   -- The CXX compiler identification is GNU 7.5.0
>   -- Detecting C compiler ABI info
>   -- Detecting C compiler ABI info - done
>   -- Check for working C compiler: /usr/bin/cc - skipped
>   -- Detecting C compile features
>   -- Detecting C compile features - done
>   -- Detecting CXX compiler ABI info
>   -- Detecting CXX compiler ABI info - done
>   -- Check for working CXX compiler: /usr/bin/c++ - skipped
>   -- Detecting CXX compile features
>   -- Detecting CXX compile features - done
>   GMP_INCLUDES=GMP_INCLUDES-NOTFOUND
>   CMake Error at /snap/cmake/870/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
>     Could NOT find GMP (missing: GMP_INCLUDES GMP_LIBRARIES GMP_VERSION_OK)
>     (Required is at least version "5.1.0")
>   Call Stack (most recent call first):
>     /snap/cmake/870/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
>     cmake/FindGMP.cmake:74 (find_package_handle_standard_args)
>     CMakeLists.txt:17 (find_package)
>   
>   
>   -- Configuring incomplete, errors occurred!
>   See also "/ipfs/tmpdir/pip-install-jgg5wz7d/chiavdf_ef617ebcf39f4fb4bd1906d66b679e00/CMakeFiles/CMakeOutput.log".
>   Traceback (most recent call last):
>     File "/service/sun/chia-blockchain/venv/lib/python3.7/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 280, in <module>
>       main()
>     File "/service/sun/chia-blockchain/venv/lib/python3.7/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 263, in main
>       json_out['return_val'] = hook(**hook_input['kwargs'])
>     File "/service/sun/chia-blockchain/venv/lib/python3.7/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 205, in build_wheel
>       metadata_directory)
>     File "/ipfs/tmpdir/pip-build-env-gfe45a03/overlay/lib/python3.7/site-packages/setuptools/build_meta.py", line 222, in build_wheel
>       wheel_directory, config_settings)
>     File "/ipfs/tmpdir/pip-build-env-gfe45a03/overlay/lib/python3.7/site-packages/setuptools/build_meta.py", line 207, in _build_with_temp_dir
>       self.run_setup()
>     File "/ipfs/tmpdir/pip-build-env-gfe45a03/overlay/lib/python3.7/site-packages/setuptools/build_meta.py", line 150, in run_setup
>       exec(compile(code, __file__, 'exec'), locals())
>     File "setup.py", line 281, in <module>
>       zip_safe=False,
>     File "/ipfs/tmpdir/pip-build-env-gfe45a03/overlay/lib/python3.7/site-packages/setuptools/__init__.py", line 153, in setup
>       return distutils.core.setup(**attrs)
>     File "/usr/lib/python3.7/distutils/core.py", line 148, in setup
>       dist.run_commands()
>     File "/usr/lib/python3.7/distutils/dist.py", line 966, in run_commands
>       self.run_command(cmd)
>     File "/usr/lib/python3.7/distutils/dist.py", line 985, in run_command
>       cmd_obj.run()
>     File "/ipfs/tmpdir/pip-build-env-gfe45a03/overlay/lib/python3.7/site-packages/wheel/bdist_wheel.py", line 299, in run
>       self.run_command('build')
>     File "/usr/lib/python3.7/distutils/cmd.py", line 313, in run_command
>       self.distribution.run_command(command)
>     File "/usr/lib/python3.7/distutils/dist.py", line 985, in run_command
>       cmd_obj.run()
>     File "/usr/lib/python3.7/distutils/command/build.py", line 135, in run
>       self.run_command(cmd_name)
>     File "/usr/lib/python3.7/distutils/cmd.py", line 313, in run_command
>       self.distribution.run_command(command)
>     File "/usr/lib/python3.7/distutils/dist.py", line 985, in run_command
>       cmd_obj.run()
>     File "setup.py", line 112, in run
>       self.build_extension(ext)
>     File "setup.py", line 139, in build_extension
>       subprocess.check_call(["cmake", ext.sourcedir] + cmake_args, env=env)
>     File "/usr/lib/python3.7/subprocess.py", line 328, in check_call
>       raise CalledProcessError(retcode, cmd)
>   subprocess.CalledProcessError: Command '['cmake', '/ipfs/tmpdir/pip-install-jgg5wz7d/chiavdf_ef617ebcf39f4fb4bd1906d66b679e00/src', '-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=/ipfs/tmpdir/pip-install-jgg5wz7d/chiavdf_ef617ebcf39f4fb4bd1906d66b679e00/build/lib.linux-x86_64-3.7', '-DPYTHON_EXECUTABLE=/service/sun/chia-blockchain/venv/bin/python', '-DCMAKE_BUILD_TYPE=Release']' returned non-zero exit status 1.
>   ----------------------------------------
>   ERROR: Failed building wheel for chiavdf
> Failed to build chiavdf
> ERROR: Could not build wheels for chiavdf which use PEP 517 and cannot be installed directly

>

**Comment by lvcivs** [comment]
Why do you want to install a timelord? It is not needed for normal use (plotting, farming).

**Reply by arde2020** [reply]
OK ,I understand I do not use the  timelord so  I ignore the problem

**Reply by arde2020** [reply]
Thanks very much
